README.md

```diff
@@ -1,9 +1,14 @@
 # cleanvision-examples
 
-Notebooks demonstrating example applications of the [cleanvision](https://github.com/cleanlab/cleanvision/) library.
+The notebooks in this repository demonstrate the example applications
+of [CleanVision](https://github.com/cleanlab/cleanvision/) on following datasets.
+- [Caltech-256](caltech256.ipynb)
+- [Caltech-UCSD Birds-200-2011 (CUB-200-2011)](cub200.ipynb)
+- [CIFAR10](cifar10.ipynb)
+- [Food-10](food101.ipynb)
```

Comment:
this should be Food-101 not 10, and also update it inside that notebook if needed